docs/Pflichtenheft.tex

```diff
@@ -29,7 +29,25 @@
 \pagebreak
 
 \section{Einleitung}
-\Blindtext[1]
+In der heutigen Zeit dringt die Vernetzung in immer neue Bereiche vor. So auch in die Industrie und ihre Fertigungsprozesse.
+Dabei soll das Projekt Industrial Data Space helfen, einen Datenraum zu erzeugen, der Datensicherheit, Datensouveränität und
+Transparenz bietet. Auf diese Weise soll dazu beigetragen werden, Industrie 4.0 voranzutreiben und somit durch Vernetzung von
+Maschinen, Computern und Menschen die Leistungsfähigkeit der Industrie zu erhöhen und auf die Zukunft auszurichten.
+Dabei hilft das auf TCP/IP basierende Kommunikationsprotokoll OPC UA, Maschienen miteinander interagieren zu lassen.
+
+Die hier vorgestellte Simulation ermöglicht es, die Vorteile der Vernetzung von Maschinen mit OPC UA zu demonstrieren und zu verdeutlichen, um Industriekunden von den neuen Möglichkeiten zu überzeugen.
+Die Hauptkomponenten sind ein Server (Fertigungssimulation) und ein Client (Überwachungskonsole),
+die jeweils eine graphische Benutzeroberfläche (GUI) besitzen und auf getrennten Maschinen laufen können.
+Sie kommunizieren lediglich über OPC UA.
+In der GUI der Fertigungssimulation wird ein industrieller Prozess dargestellt.
+Drei Tanks werden mit je einer farbigen Flüssigkeit gefüllt und leiten diese in einen weiteren Tank,
+wo die Flüssigkeiten durch einen Motor gemischt werden. Daten und Werte des Prozesses werden ebenfalls angezeigt. Dazu gibt es
+die Möglichkeit, die Produktion zu konfigurieren, wie zum Beispiel Zu- und Abflussmengen oder Grenzen für Füllstände.
```

Comment:
Ich finde der Satz hört sehr überraschend auf, als würde am Ende noch ein Verb fehlen. Mein Vorschlag:
"...die Möglichkeit, die Produktion zu konfigurieren, zum Beispiel in dem man Zu- und Abflussmengen oder Grenzen für Füllstände einstellt."